lib/src/ChatPane.dart

```diff
@@ -93,20 +93,7 @@ class ChatPane {
       DivElement userlistitem = document.querySelector(".users-list");
       HtmlElement avatar = userlistitem.querySelector("[title='"+e.newMember.id+"']");
       ImageElement picture = avatar.parent.querySelector(".user-list-avatar");
-      switch (e.newMember.status){
-      case "dnd":
-      picture.style.borderColor="red";
-      break;
-      case "online":
-      picture.style.borderColor="green";
-      break;
-      case "idle":
-      picture.style.borderColor="orange";
-      break;
-      case "offline":
-      picture.style.borderColor="grey";
-      break;
-    }
+      picture.id=e.newMember.status;
```

Comment:
It now uses a HTML id attribute. Normally you expect only one of them to exist on a page. The proper way would to add/set the class name and reference the class name from css using the selector `.(classname)` eg. `.dnd`
Reference:
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id
https://developer.mozilla.org/en-US/docs/Web/API/Element/className